Bottom: d10041baeb6166d07ebb9c03eea081b65d5491a3
Top:    6e364f6e6b3397780c5b7f9eb3dd7ca6782dc20b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 01:33:10 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
